.travis.yml

```diff
@@ -11,6 +11,7 @@ install: bundle install --without=development
 
 rvm:
   - rbx-2
+  - rbx-3.37
```

Comment:
Yes, I've been trying to explain on several instances of this I've noticed. It's just plain `rbx` -- there's a script for testing this, @brixen put together: https://github.com/rubinius/travis-canary